libazureinit/src/config.rs

```diff
@@ -181,6 +181,10 @@ impl Default for AzureProxyAgent {
     }
 }
 
+/// Global default timeout for Wire server connections.
+pub const DEFAULT_WIRESERVER_TOTAL_RETRY_TIMEOUT_SECS: f64 = 1200.0;
+/// Global default retry interval for Wire server requests.
+pub const DEFAULT_WIRESERVER_CONNECTION_TIMEOUT_SECS: f64 = 2.0;
```

Comment:
```suggestion
/// Retry wireserver up to 20 minutes.  The VM has most likely failed provisioning
/// due to timeout at this point.
pub const DEFAULT_WIRESERVER_TOTAL_RETRY_TIMEOUT_SECS: f64 = 1200.0;
pub const DEFAULT_WIRESERVER_CONNECTION_TIMEOUT_SECS: f64 = 60.0;
pub const DEFAULT_WIRESERVER_READ_TIMEOUT_SECS: f64 = 60.0;
```

Reply:
I don't know that these need forward definitions, there should be no reference to this outside of config?